frontend/src/metabase/admin/settings/components/EmbeddingSettings/EmbeddingSdkSettings.tsx

```diff
@@ -204,6 +204,7 @@ export function EmbeddingSdkSettings({
           />
           <SetByEnvVarWrapper setting={sdkOriginsSetting}>
             <SettingTextInput
+              data-testid="sdk-origins-input"
```

Comment:
Is this used? I believe it's here because you extract the test to another PR, but I'd rather we bundle this change with the test so we know where it's used.

Reply:
nope i forgot to remove it